controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1855,6 +1855,23 @@ func (r *BareMetalHostReconciler) createHostFirmwareSettings(info *reconcileInfo
 	return nil
 }
 
+func (r *BareMetalHostReconciler) createHostUpdatePolicy(info *reconcileInfo) error {
```

Comment:
I spent some time trying to figure this out but haven't gotten far. createHostFirmwareSettings()/createHostFirmwareComponents() are called in registerHost() so that was my first thought but since we're not creating only setting owner probably not a good idea. Reconciler doesn't have reconcileInfo readily available. I'd appreciate a pointer or two.

Reply:
Don't we need a reconciler for hostUpdatePolicy ?

Reply:
OK I added a call from registerHost() to hostUpdatePolicySetOwnerReference().